test/com/minseungkwon/datastructure/DynamicProgramming/DPAlgorithmTest.java

```diff
@@ -0,0 +1,18 @@
+package com.minseungkwon.datastructure.DynamicProgramming;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class DPAlgorithmTest {
+    @Test
+    public void test_minCostClimbingStairs() {
```

Comment:
can we have a implementation which does not use dp and compare their run time by using 
https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis() ?

Reply:
Added the new method which is not using DP. 
Also calculated the runtimes of both solutions, but it showed the same time gaps which is 1.

Reply:
I guess the test set is not big enough to show the difference of using DP. As long as now you know how to check actual runtime, I think it is good :) 👍